src/main/java/uk/co/westhawk/snmp/stack/AsnEncoderv3.java

```diff
@@ -62,11 +62,17 @@
  * <http://www.gnu.org/licenses/lgpl-3.0.html>.
  * ╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱
  */
-import uk.co.westhawk.snmp.util.*;
-import java.io.*;
-import java.nio.ByteBuffer;
-import java.util.*;
-import java.util.stream.IntStream;
+
+import uk.co.westhawk.snmp.util.SnmpUtilities;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Enumeration;
+
+import static uk.co.westhawk.snmp.util.SnmpUtilities.computeFingerprint;
```

Comment:
I would use `SnmpUtilities.<methodName>` for better readability. Please remove these imports.